definitions/artifacts/mongodbatlas-authentication.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$md": {
+      "access": "public",
+      "defaultTargetConnectionGroup": "credentials",
+      "defaultTargetConnectionGroupLabel": "MongoDB Atlas Credentials",
+      "importing": {
+        "group": "authentication"
+      },
+      "diagram": {
+        "isLinkable": false
+      },
+      "name": "mongodbatlas-credentials"
```

Comment:
Name doesn’t match file name. They should both be whatever the cloud vendor refers to them Ass “credentials” or “authentication” “service acct” etc

Reply:
they call them api-keys, but its a public/private key pair.